src/beautifiers/js-beautify.coffee

```diff
@@ -9,7 +9,6 @@ module.exports = class JSBeautify extends Beautifier
     XML: true
     Handlebars: true
     Mustache: true
-    Marko: true
```

Comment:
Does `js-beautify` handle Marko language well enough? If so, then leaving Marko support in JS-Beautify is a good idea, so users can choose and switch between them. We then set `defaultBeautifier` for the language to `Marko Beautifier` instead.